packages/liveblocks-react/src/__tests__/_restMocks.ts

```diff
@@ -7,7 +11,7 @@ export function mockGetThreads(
     RestRequest<never, never>,
     RestContext,
     {
-      data: ThreadDataPlain<any>[];
+      data: ThreadData<any>[];
```

Comment:
This is wrong. We return plain data from the server so we should not adapt our mocks to make it easier to test.

Created a new issue so we can tackle that later https://github.com/liveblocks/liveblocks.io/issues/1850

Reply:
Thanks for pointing this out. I assumed any `Date` object would be automatically converted to strings in the 8601 format during serialization, so the returned date would indeed be plain. But, I can see how we could miss properly serializing other properties in the future.